frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -141,6 +143,13 @@ watch(
                         v-bind="field"
                         class="w-100 custom-height"
                         @input="cleanupForestClientNumberInput()"
+                        @keypress.enter="
+                            field.value &&
+                                !errorMessage &&
+                                verifyForestClientNumber(
+                                    forestClientNumbersInput
+                                )
+                        "
```

Comment:
Thanks Jota, yes, if no error then the data can be submit to backend.

This is interesting, I was trying to see how "errorMessage" is produced and what's contains in there.
I thought the vee-validate validation schema is in this ForestClientInput.vue but I could not find it. Instead, it is at the utils.ts and I don't know how it validate the FORM from parent component to child components. But that's just I don't have too much experience with vee-validate. It's working but in a hidden way.

* could you align the conditions vertically?
* since this "**forestClientNumbersInput**" is a ref, probably no need to pass it. But that's ok, fine with that.
* In "verifyForestClientNumber()" method, I am not sure the validation for line: 38~42 is for. Are we expecting these number (in array) can still be string Not-A-Number? (I thought the vee-validate schema already checks). Please check if this is unnecessary.
* I believe Vee-validate also can add some validation messages when needed, you guys might want to utilize that next time for displaying error messages; but I don't know how easy it is. Just a suggestion, not for this PR.

Reply:
Hello Ian! I'll align the code. The "**verifyForestClientNumber()**" method check if the array contain only valid numbers, I'm not sure if veeValidate does this kind of comparison but I'll take a look.

Reply:
VeeValidate does check to if there are only numbers and the button is disabled if not, so I think it can be taken out

Reply:
oh Ian, sorry just saw the message, the validation schema is from the form level, so it's passed from GrantAccess and grant delegated admin component 
<img width="1018" alt="Screen Shot 2024-05-13 at 3 35 05 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/61b34dc9-3222-4c4d-913f-c9caa2a677fe">